src/main.py

```diff
@@ -98,18 +98,23 @@
     # such as unsafe-inline or unsafe-eval is highly discouraged, as these
     # may lead to XSS attacks.
     'csp_policy': {
-        # https://developers.google.com/fonts/docs/technical_considerations
-        'font-src':    '\'self\' themes.googleusercontent.com '
-                       '*.gstatic.com',
+        # Fallback.
+        'default-src': '\'self\'',
+        # Disallow Flash, etc.
+        'object-src': '\'none\'',
+        # Google Analytics.
+        'script-src': '\'self\' https://www.google-analytics.com/analytics.js',
         # Maps, YouTube provide <iframe> based embedding at these URIs.
-        'frame-src':   '\'self\' www.google.com www.youtube.com',
-        # Assorted Google-hosted APIs.
-        'script-src':  '\'self\' *.googleanalytics.com *.google-analytics.com',
+        'child-src':  '\'self\' https://www.google.com https://www.youtube.com',
+        # Deprecated. Used for supporting browsers that use CSP 1.0 only.
+        'frame-src':  '\'self\' https://www.google.com https://www.youtube.com',
         # In generated code from http://www.google.com/fonts
-        'style-src':   '\'self\' fonts.googleapis.com *.gstatic.com',
-        # Fallback.
-        'default-src': '\'self\' *.gstatic.com',
-        'report-uri':  '/csp',
+        'style-src':  '\'self\' https://fonts.googleapis.com '
```

Comment:
Right, the current set of browsers supporting CSP don't "support" style/expression based XSS.
What still would be possible are "scriptless attacks" like style defacement or other attacks described [here](http://www.nds.rub.de/media/emma/veroeffentlichungen/2012/08/16/scriptlessAttacks-ccs2012.pdf).
I removed 'unsafe-inline' from style-src again. If developers want to use inline styles they can enable it manually.